src/diem/testing/miniwallet/app/static/openapi.yaml

```diff
@@ -1,69 +1,64 @@
-        * [Send payment endpoint](#post-/accounts/-account_id-/payments): required for test sending payment.
-        * [Get KYC sample endpoint](#get-/kyc_sample): required for test sending payment equal / above travel rule threshold limit.
-
-      You can selectively implement endpoints for testing sub-set features. For example,
-      [get KYC sample endpoint](#get-/kyc_sample) is not required until you need test
-      payment that triggers travel rule.
-
-
-      The **MiniWallet Test Suite** also provides tests to verify teh **MiniWallet API** built
-      for your application is meeting requirements for running payment integration tests.
-
-
-      We will improve and grow **MiniWallet Test Suite** to cover more cases.
   license:
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0.html
   version: 0.0.1
 tags:
-  - name: Minimum
+  - name: Test sending and receiving payments on-chain
     description: >
-        Minimum required endpoints for running **MiniWallet Test Suite** to test sending
-        and receiving payment under travel rule threshold limit.
-  - name: Off-chain
+        The following are the minimum required endpoints for running the Diem MiniWallet Test Suite
+        to test sending and receiving payments under the Travel Rule threshold limit. 
+  - name: Test sending and receiving payments off-chain
     description: >
-        Required endpoint for running **MiniWallet Test Suite** to test payment requires
-        Diem off-chain API, e.g. payment amount is equal or above travel rule threshold limit.
-  - name: Optional
+        To test transactions that use the Diem off-chain API, you will need to use the following
+        required endpoint. 
+  - name: debug
     description: >
-        Optional debug endpoint that is not required to implement for running
-        **MiniWallet Test Suite**
+        This is an optional endpoint you can use for debugging. 
+
 paths:
   /accounts:
     post:
       summary: Create a new account
       description: >
-          * When the account is required to provide KYC data during off-chain KYC exchange process,
-            server should use `kyc_data` property value as KycDataObject and send to counterparty service.
-          * Server should not raise error for the case `kyc_data` property value is valid but
-            it does not meet its business criterias.
-          * `balances` property values are the initial deposit to the account.
-          * Client should call [get account balances endpoint](#get-/accounts/-account_id-/balances)
-            to check account balances after the account is created.
-          * Client should store the response account id for all
-            the operations to the account.
-          * MiniWallet Test Suite isolates test data by creating new account for
-            each test case.
+          This method creates a new account. 
```

Comment:
add one blank line here, it is required to render the following bullets correctly.